apps/projects/app/components/Panel/FundIssues/FundIssues.js

```diff
@@ -63,6 +63,7 @@ const bountiesFor = ({ bountySettings, issues, tokens }) => issues.reduce(
       slotsIndex: 0,
       payout: issue.payout || 0,
       token: tokens.find(t => t.symbol === issue.symbol) || tokens[0],
+      openSubmission: issue.openSubmission,
```

Comment:
We are setting this on a per-issue basis, but in the form we are setting it globally, for all issues-being-funded. Is that correct?

Do we use this version of `openSubmission`? Can we remove it?

Reply:
Removed.